config/.env

```diff
@@ -0,0 +1,2 @@
+DATABASE_URL=postgresql://localhost/blockchain
+GOOGLE_MAPS_API_KEY=
```

Comment:
I'd just remove this.. to avoid it being in the environment as an empty string which would cause a blowup

Reply:
OK. Just hate to forget about it.